packages/webdriverio/webdriverio.d.ts

```diff
@@ -34,16 +34,13 @@ declare namespace WebdriverIO {
             <T>(script: string | ((...arguments: any[]) => T), ...arguments: any[]): Promise<T>;
         };
 
-        // there is no way to add callback as last parameter after `...args`.
```

Comment:
Can we also remove this comment about the callback in `packages/wdio-sync/webdriverio.d.ts`?